documentation/operators/src/nodes/network-requester-setup.md

```diff
@@ -1,6 +1,10 @@
 # Network Requesters
 
-> The Nym gateway was built in the [building nym](../binaries/building-nym.md) section. If you haven't yet built Nym and want to run the code, go there first.
+> The Nym network requester was built in the [building nym](../binaries/building-nym.md) section. If you haven't yet built Nym and want to run the code, go there first.
+
+```admonish info
+As a result of [Project Smoosh](../faq/smoosh-faq.md), the current version of `nym-network-requester` can be ran as a part of [`nym-gateway`](./gateway-setup.md#initialising-gateway-with-network-requester). This combination is a basis of Nym exit gateway node - an essential piece in our new setup. Please read more in our [Project Smoosh FAQ](../faq/smoosh-faq.md) and [Exit Gateways Page](../legal/exit-gateway.md). We recommend operators to start shifting to that model instead of running gateway and network requester as two separated binaries.
```

Comment:
Also here - lets bring the phrase 'exit gateway' in and start cementing it :)

Reply:
Also see above comment on rephrasing last sentence from gw page

Reply:
i go with exit gateway everywhere to entrench it.